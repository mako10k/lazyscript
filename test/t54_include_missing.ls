{- #include "test/does_not_exist.ls" -}
()
