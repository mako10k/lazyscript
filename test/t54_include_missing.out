<ROOT>/test/t54_include_missing.ls:1.1-40: error: cannot open include file 'test/does_not_exist.ls'
